### PR TITLE
Fix for error: Network not found: no network with matching name 'default'

### PR DESCRIPTION
**Pull request by kishansag**
Fix for error: Network not found: no network with matching name 'default'

Related to: https://github.com/kubernetes/minikube/issues/828

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
@kishansag  We have a policy where we file issues for each patch. Refer https://github.com/minishift/minishift/blob/master/CONTRIBUTING.adoc#contributing-patches . Issues are permanent but pull requests are temporary and contributors can  create or close multiple pull requests. So please open an issue and  include it in the commit message e.g. "Issue #xyz Documentation for KVM network not found issue".

Also please squash the commits in to just one commit.

File: docs/source/getting-started/setting-up-driver-plugin.adoc [hunk 1]
```diff
@@ -33,6 +33,12 @@ $ sudo chmod +x /usr/local/bin/docker-machine-driver-kvm
 
 For more information, see the GitHub documentation of the link:https://github.com/dhiltgen/docker-machine-kvm#quick-start-instructions[docker machine KVM driver].
 
+KVM has a standing issue. You might face an error like
```
Comment (budhrg) [inline docs/source/getting-started/setting-up-driver-plugin.adoc]: Should be place at troubleshooting page? @LalatenduMohanty

File: docs/source/getting-started/setting-up-driver-plugin.adoc [hunk 2]
```diff
@@ -33,6 +33,12 @@ $ sudo chmod +x /usr/local/bin/docker-machine-driver-kvm
...
+----
+error: Network not found: no network with matching name 'default'
+----
+If so, start a default network using virt-manager(virt-manager > edit > connection details > virtual networks > default), more link:https://github.com/kubernetes/minikube/issues/828[here]
+
```
Comment (LalatenduMohanty) [inline docs/source/getting-started/setting-up-driver-plugin.adoc]: This should go the [troubleshooting guide](https://docs.openshift.org/latest/minishift/troubleshooting/troubleshooting-driver-plugins.html) i.e. https://github.com/minishift/minishift/blob/master/docs/source/troubleshooting/troubleshooting-driver-plugins.adoc

File: docs/source/getting-started/setting-up-driver-plugin.adoc (hunk 1 above)
Comment (gbraad) [reply docs/source/getting-started/setting-up-driver-plugin.adoc]: standing issue?

File: docs/source/getting-started/setting-up-driver-plugin.adoc (hunk 2 above)
Comment (gbraad) [reply docs/source/getting-started/setting-up-driver-plugin.adoc]: s/a/the
```diff
- virt-manager(virt-manager > edit > connection details > virtual networks > default)
+virt-manager (virt-manager > edit > connection details > virtual networks > default)
```
s/more here/For more informatioon, see link:...[...]

Would it be possible to refer to a real resource that explains how to active the default network, or how to restore it?

I would also prefer to see instructions from the command line when possible, such as:
```
virsh net-undefine default
virsh net-destroy default
virsh net-list
virsh net-define /usr/share/libvirt/networks/default.xml
virsh net-start default
virsh net-autostart default
```

**Comment by praveenkumar** [comment]
@kishansag any plan to update this PR as per reviews?

**Comment by kishansag** [comment]
@praveenkumar Sorry, I have been a little busy, so this was slightly ignored. I will create a new issue about this and make a PR.

**Pull request closed without merging**